app/services/rails_engine_service.rb

```diff
@@ -0,0 +1,30 @@
+class RailsEngineService
+  class << self
+    def get_data(endpoint)
+      conn = Faraday.new(url: "http://localhost:3000/api/v1/")
```

Comment:
make your connection just to http://localhost:3000. `api/v1` is technically part of the uri still.